integration_tests/testsuite/deploy_app.py

```diff
@@ -102,37 +157,56 @@ def deploy_app(base_image, builder_image, appdir, yaml):
         if builder_image:
             _set_builder_image(builder_image)
 
-        deployed_version = test_util.generate_version()
+        if environment == 'GAE':
```

Comment:
Can you make these named constants, and change 'GAE' to 'GAE_FLEX'?  It's conceivable that we could add GAE Standard some day (post-Titanium).  Also GCF.